econml/inference/_inference.py

```diff
@@ -462,9 +471,12 @@ def fname_transformer(x):
         else:
             def fname_transformer(x):
                 return x
+        # d_t=1 here since we measure the effect across all Ts
+        # A dummy treatment names is passed in
```

Comment:
Consider rewording, since it is not actually passed in

```suggestion
        # Omit treatment name argument to fall back to the default
```